src/components/multi-action-button/multi-action-button.spec.tsx

```diff
@@ -672,4 +672,30 @@ describe("MultiActionButton", () => {
       expect(tempWrapper.contains("Some string")).toBe(true);
     });
   });
+
+  describe("when customWidth is passed", () => {
+    it("justify-content on StyledButton set to space-between", () => {
+      wrapper = render({ customWidth: "100%" }, mount);
```

Comment:
we should test the `width` prop now there is no `customWidth` prop - this probably won't even pass